packages/flutter/test/cupertino/checkbox_test.dart

```diff
@@ -409,9 +409,320 @@ void main() {
       paints
         ..drrect(
           color: const Color(0xfff44336),
-          outer: RRect.fromLTRBR(13.0, 13.0, 31.0, 31.0, const Radius.circular(5)),
-          inner: RRect.fromLTRBR(17.0, 17.0, 27.0, 27.0, const Radius.circular(1)),
+          outer: RRect.fromLTRBR(15.0, 15.0, 29.0, 29.0, const Radius.circular(5)),
+          inner: RRect.fromLTRBR(19.0, 19.0, 25.0, 25.0, const Radius.circular(1)),
+        ),
+    );
+  });
+
+  testWidgets('Checkbox default colors, and size in light mode', (WidgetTester tester) async {
```

Comment:
What do you think if we just use a golden test, which tests the screenshot? I think this way the test will be easier to maintain.
(In case you're not familiar with golden tests, here's an example: https://github.com/flutter/flutter/blob/3a737c9a5688319717af096618a00ddc734a703a/packages/flutter/test/cupertino/action_sheet_test.dart#L47 )

Reply:
That's a great idea. I think I can replace the general 'default colors, shape, size, etc' tests with goldens.